src/CONST.ts

```diff
@@ -134,7 +134,7 @@ const onboardingEmployerOrSubmitMessage: OnboardingMessage = {
                 '\n' +
                 'Here’s how to set up your bank account:\n' +
                 '\n' +
-                '1. Click your profile picture.\n' +
+                '1. Click the Settings tab.\n' +
```

Comment:
Its lowercase in all the other prompts
```suggestion
                '1. Click the settings tab.\n' +
```